### PR TITLE
#AV-64697 Added example to use with SDk with new Patch Syntax

### PR DESCRIPTION
**Pull request by shounak-opcito**
Added examples for the following operations using latest Patch api. (One  can find the example function in file patch_api_example.py):
- add new server to existing pool (add_server_to_pool())
- remove server from a pool (remove_server_from_pool())
- update server in existing pool (update_server_in_pool())

### CODE REVIEW

File: python/avi/sdk/samples/patch_api_example.py [hunk 1]
```diff
@@ -0,0 +1,168 @@
...
+    2. Add new servers to existing pool
+    3. Replace a server in the pool
+    4. Delete a server from the pool
+    5. Delete a pool
+    """
+    def __init__(self, api_session):
+        self.api_session = api_session
+        self.api_util = ApiUtils(api_session)
+
+    def get_server_obj(self, server):
```
Comment (grastogi23) [inline python/avi/sdk/samples/patch_api_example.py:22]: Please document functions use 
https://avinetworks.atlassian.net/wiki/spaces/EN/pages/12917148/Coding+Guidelines

File: python/avi/sdk/samples/patch_api_example.py [hunk 2]
```diff
@@ -0,0 +1,168 @@
...
+        """
+        Get server objects from list of raw server string
+        """
+        servers_obj = []
+        for server in servers:
+            servers_obj.append(self.get_server_obj(server))
+        return servers_obj
+
+    def create_pool(self, name, servers_obj, monitor_names=None,
+                    lb_algorithm='LB_ALGORITHM_LEAST_CONNECTIONS'):
```
Comment (grastogi23) [inline python/avi/sdk/samples/patch_api_example.py:49]: https://avinetworks.atlassian.net/wiki/spaces/EN/pages/12917148/Coding+Guidelines

File: python/avi/sdk/samples/patch_api_example.py [hunk 3]
```diff
@@ -0,0 +1,168 @@
...
+            'health_monitor_refs': health_monitors
+        }
+        resp = self.api_session.post('pool', data=json.dumps(pool_obj))
+        if resp.status_code in range(200, 299):
+            return resp
+        else:
+            print('Error in creating pool :%s' % resp.text)
+            exit(0)
+
+    def add_server_to_pool(self, server, pool_uuid):
```
Comment (grastogi23) [inline python/avi/sdk/samples/patch_api_example.py:81]: https://avinetworks.atlassian.net/wiki/spaces/EN/pages/12917148/Coding+Guidelines

File: python/avi/sdk/samples/patch_api_example.py [hunk 4]
```diff
@@ -0,0 +1,168 @@
...
+        resp = self.api_session.patch(path=api_url, data=json.dumps(data))
+        if resp.status_code in range(200, 299):
+            return resp
+
+        else:
+            print("Error in updating server in pool: %s" % resp.text)
+            exit(0)
+
+
+if __name__ == "__main__":
```
Comment (grastogi23) [inline python/avi/sdk/samples/patch_api_example.py:159]: Add examples section for this. Look at how f5 converters do it.

**Review by grastogi23 — CHANGES_REQUESTED** [review]

File: python/avi/sdk/samples/patch_api_example.py (hunk 1 above)
Comment (shounak-opcito) [reply python/avi/sdk/samples/patch_api_example.py:22]: added required comment

File: python/avi/sdk/samples/patch_api_example.py (hunk 2 above)
Comment (shounak-opcito) [reply python/avi/sdk/samples/patch_api_example.py:49]: added required comment

File: python/avi/sdk/samples/patch_api_example.py (hunk 3 above)
Comment (shounak-opcito) [reply python/avi/sdk/samples/patch_api_example.py:81]: added required comment

File: python/avi/sdk/samples/patch_api_example.py (hunk 4 above)
Comment (shounak-opcito) [reply python/avi/sdk/samples/patch_api_example.py:159]: Added example section

**Review by grastogi23 — APPROVED** [review]

**Pull request merged**